tests/suites/spaces_ec2/upgrade_charm_with_bind.sh

```diff
@@ -17,7 +17,7 @@ run_upgrade_charm_with_bind() {
     juju_machine_id=$(juju show-machine --format json | jq -r '.["machines"] | keys[0]')
 
     # Deploy test charm to dual-nic machine
-    juju deploy cs:~juju-qa/bionic/space-defender-0 --bind "defend-a=alpha defend-b=isolated" --to "${juju_machine_id}"
+    juju deploy cs:~juju-qa/focal/space-defender-2 --bind "defend-a=alpha defend-b=isolated" --to "${juju_machine_id}" --force
```

Comment:
space-defender-0 and -1 work on bionic only. I pushed a -2 and -3 with focal and groovy support.